views/clients/show.erb

```diff
@@ -1,53 +1,63 @@
-<div class="container">
-<h2> Client: <%= @client.name %>
-<a href="/clients/<%= @client.id %>/edit">edit</a>
+<div class='container'>
+<h2> Client: <%= @client.name %> -
+<a href="/clients/<%= @client.id %>/edit"> Edit</a>
 </h2>
 <p>
-    Name: <%= @client.contact_name %> <br>
-    Email: <%= @client.contact_email %> <br>
-    Description: <%= @client.description %> <br>
+    <b>Name:</b> <%= @client.contact_name %> <br>
+    <b>Email:</b> <%= @client.contact_email %> <br>
+    <b>Description:</b> <%= @client.description %> <br>
     <%= @client.active ? 'Active' : 'Inactive' %><br>
     <a href="/clients/<%= @client.id %>/billing"> <h4>Billing</h4> </a>
 </p>
+<br>
 
 <% if @projects.empty? %>
   <p>
     <%= @client.name %> has no projects
   </p>
 <% else %>
   <% if @client.name != 'default' %>
-    <% Report.plugin_matrix.each do |resource_type, resource_measurements| %>
-      <table class="table">
-        <thead>
-          <h4><%= resource_type.capitalize%></h4>
-          <th>Name</th>
-          <%resource_measurements.each do |header| %>
-            <th><%= header %></th>
-          <% end %>
-        </thead>
-        <tbody>
-          <% @client_data.each do |project_name, project_resource|%>
-            <% project_resource.each do |resource|%>
-              <% resource[resource_type][0].each do |name, res_hash| %>
+    <% @client_data.each do |project_name, array_hash| %>
+      <% array_hash[0].each do |resource_type, resource_hash| %>
+        <table class='table'>
+          <h4><%= resource_type == 'db' ? 'Database' : resource_type.capitalize %></h4>
+          <thead>
+            <% @keys = resource_hash[0].values[0].keys - ['id'] %>
+            <tr>
+              <th>Name</th>
+              <% @keys.each do |k| %>
+                <% if k == 'DBSize' %>
+                  <th>Database Size</th>
+                <% elsif k == 'VCPUCount' %>
+                  <th>CPU Count</th>
+                <% else %>
+                  <th><%= k.underscore.humanize.titleize %></th>
```

Comment:
I ran into the following [error](https://github.com/osuosl/iam/files/1062633/NoMethodError.at._clients_3.pdf) using imported production data.